docker/e2e/Dockerfile.machine

```
@@ -0,0 +1,50 @@
+
+RUN useradd --uid 1001 --create-home --shell /bin/bash agent
+USER agent
+WORKDIR /home/agent
+
+ENV BUN_INSTALL=/home/agent/.bun
+ENV PATH="/home/agent/.bun/bin:/home/agent/.npm-global/bin:/home/agent/.local/bin:${PATH}"
+
+RUN mkdir -p /home/agent/.npm-global \
+ && npm config set prefix /home/agent/.npm-global
+
+RUN curl -fsSL https://bun.sh/install | bash
```

Hadolint (2.14.0)

[warning] Set the SHELL option -o pipefail before RUN with a pipe in it. If you are using /bin/sh in an alpine image or if your shell is symlinked to busybox then consider explicitly setting your SHELL to /bin/ash, or disable this check

(DL4006)